photutils/isophote/model.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import sys
+import numpy as np
+from scipy.interpolate import LSQUnivariateSpline
```

Comment:
in addition of moving these imports, the tests that use any code relying on scipy also need to be decorated with ``@pytest.mark.skipif('not HAS_SCIPY')`` where ``HAS_SCIPY`` is defined as 

```
try:
    import scipy
    HAS_SCIPY = True
except ImportError:
    HAS_SCIPY = False
```

e.g.: photutils/segmentation/tests/test_core.py